controllers/databaseclusterbackup_controller.go

```diff
@@ -185,6 +180,32 @@ func (r *DatabaseClusterBackupReconciler) Reconcile(ctx context.Context, req ctr
 	return ctrl.Result{Requeue: requeue}, nil
 }
 
+func (r *DatabaseClusterBackupReconciler) reconcileMeta(
+	ctx context.Context,
+	backup *everestv1alpha1.DatabaseClusterBackup,
+	cluster *everestv1alpha1.DatabaseCluster,
+) error {
+	var needUpdate bool
+	if len(backup.ObjectMeta.Labels) == 0 {
+		backup.ObjectMeta.Labels = map[string]string{
+			databaseClusterNameLabel: backup.Spec.DBClusterName,
+			fmt.Sprintf(backupStorageNameLabelTmpl, backup.Spec.BackupStorageName): backupStorageLabelValue,
+		}
+		needUpdate = true
+	}
+	if len(backup.ObjectMeta.OwnerReferences) == 0 {
+		if err := controllerutil.SetControllerReference(cluster, backup, r.Client.Scheme()); err != nil {
+			return err
+		}
+		needUpdate = true
```

Comment:
nitpick: I think we want to check the controller, as there can can multiple non-controlling owners (probably not in our case though, I'm just to be semantically correct..)
```suggestion
	if metav1.GetControllerOf(backup) == nil {
		if err := controllerutil.SetControllerReference(cluster, backup, r.Client.Scheme()); err != nil {
			return err
		}
		needUpdate = true
```